include/zephyr/logging/log_core.h

```diff
@@ -290,7 +290,7 @@ static inline char z_log_minimal_level_to_char(int level)
 		break; \
 	} \
 	/* For instance logging check instance specific static level */ \
-	if (_inst & !IS_ENABLED(CONFIG_LOG_RUNTIME_FILTERING)) { \
+	if (_inst && !IS_ENABLED(CONFIG_LOG_RUNTIME_FILTERING)) { \
```

Comment:
Was this a bug? Maybe move this to a new PR?

Reply:
Good point. Opened #64170. I will remove this commit from that PR.